src/giveaways.js

```diff
@@ -407,6 +489,7 @@ async function enterVideoGiveaway(page) {
  */
 async function enterGiveaways(page, pageNumber) {
 	//loop through each giveaway item
+	await sqlite.open('./gg.db');
```

Comment:
I don't know how much it hurts, but I'd assume opening and closing the db connection every page probably doesn't help anything? From a little googling it sounds like opening can be expensive, so it might hurt performance a little over time. I think it would be fine to open and close the connection once in index.js.